app/src/app/shortcuts.js

```diff
@@ -2,7 +2,8 @@ import {globalShortcut, BrowserWindow} from 'electron'
 
 const shortcuts = {
   'Alt+CmdOrCtrl+I': () => {
-    BrowserWindow.getFocusedWindow().toggleDevTools()
+    const win = BrowserWindow.getFocusedWindow()
+    win && win.toggleDevTools()
```

Comment:
I think this wouldn't pass a linter, I can't recall exactly why but this was a bad style.
if (win) win.... is better